### PR TITLE
Only show draft AA on draft projects list

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: schema/project.js [hunk 1]
```diff
@@ -305,7 +305,12 @@ class Project extends BaseModel {
       .leftJoinRelation('licenceHolder')
       .withGraphFetched('[licenceHolder, additionalEstablishments(constrainAAParams), establishment(constrainEstParams)]')
       .modifiers({
-        constrainAAParams: builder => builder.select('id', 'name', 'projectEstablishments.status'),
+        constrainAAParams: builder => {
+          builder.select('id', 'name', 'projectEstablishments.status');
+          if (status !== 'inactive') {
```
Comment (lennym) [inline schema/project.js:310]: Is this really confusingly describing draft project statuses and not thing described in the UI as `Inactive`?

Comment (wheelsandcogs) [reply schema/project.js:310]: Yeah, this is value passed when the draft tab is selected - the "Inactive" tab sends `inactive-statuses`

Comment (lennym) [reply schema/project.js:310]: Could we maybe add a unit test?

Comment (wheelsandcogs) [reply schema/project.js:310]: Done.

**Pull request merged**